client/extensions/woocommerce/app/settings/payments/payment-method-item.js

```diff
@@ -67,9 +71,43 @@ class PaymentMethodItem extends Component {
 		this.props.changePaymentMethodField( this.props.site.ID, field, value );
 	}
 
-	onSave = () => {
+	onEnableHandler = ( e ) => {
 		const { method, site, translate } = this.props;
+		const enabled = e.target.value === 'yes';
+		this.props.changePaymentMethodEnabled(
+			site.ID,
+			method.id,
+			enabled
+		);
+		const successAction = () => {
```

Comment:
Minor, but I'd add a blank line before this since it really belongs more with the save below